media/components/sdfv/index.html

```diff
@@ -140,18 +144,16 @@ <h5 id="info-title"></h5>
         vscode = acquireVsCodeApi();
         message_handler = new MessageHandler();
         var debug_draw = false;
+        var showBps = false;
 
         function setRendererContent(sdfg, previewing = false,
-                                    prevent_refreshes = false) {
+            prevent_refreshes = false) {
             parsed_sdfg = daceParseSDFG(sdfg);
 
             if (daceRenderer) {
                 daceRenderer.set_sdfg(parsed_sdfg);
             } else {
-                daceInitSDFV(null, null, debug_draw);
-                if (parsed_sdfg !== null)
-                    daceRenderer = new VSCodeRenderer(parsed_sdfg, document.getElementById('contents'),
-                        daceMouseEvent, null, debug_draw);
+                daceInitSDFV(parsed_sdfg, null, debug_draw);
```

Comment:
This situation gets cleaned up using the `VSCodeRenderer` in a currently open PR from my side as well. I am in favor of keeping the `VSCodeRenderer` in, despite being able to register overlays without it now. There is a comparatively large number of functions in the renderer at this point, that are exclusive and specific to VSCode. They're often guarded by `if (vscode)` clauses etc., but I would prefer if all that functionality would live only inside VSCode. We are discussing a migration of the webclient to an `npm` package, which could further clean this up, but for now I vote on keeping this in. Thoughts?

Reply:
Sounds good to me! I will add the VSCodeRenderer back, but remove the Overlay functionalities which arent needed anymore like on_mouse_event.